pkg/test/pods.go

```diff
@@ -143,13 +143,19 @@ func Pod(overrides ...PodOptions) *v1.Pod {
 			Phase:      options.Phase,
 		},
 	}
-	if options.InitImage != "" {
-		p.Spec.InitContainers = []v1.Container{{
-			Name:      RandomName(),
-			Image:     options.InitImage,
-			Resources: options.InitResourceRequirements,
-		}}
+	if options.Overhead != nil {
+		p.Spec.Overhead = options.Overhead
 	}
+	if options.InitContainers != nil {
+
+		for _, init := range options.InitContainers {
+			init.Name = RandomName()
+			init.Image = "pause"
```

Comment:
A couple comments here. We shouldn't hardcode this to a `pause` image. This doesn't cause issues if we're just testing sidecar containers but if we're testing non-sidecar init containers this will cause the pod to be stuck initializing since the `pause` image never completes. I don't see an issue with using this as a default, but only if the image is not already specified. Also, we should use the same pause image we default the main container to, `public.ecr.aws/eks-distro/kubernetes/pause:3.2`.

Reply:
What if we just passed the initContainers through as-is and just had a method or constant called `DefaultImage` or `DefaultInitImage` instead